### ISSUE TITLE
Holo/Material design

### ISSUE THREAD

**Issue by mars3142** [nolanlawson/Catlog#40]
Will this app be updated to use Holo and/or Material Design?

**Comment by thevirus20** [comment]
After compiling with target 22 in android studio its looks great.
Thanks for a great app.

![Lollipop](http://i.imgur.com/o4tkgvw.png)

**Comment by damccull** [comment]
@nolanlawson Are you still actively maintaining this app? I realize it probably doesn't need much in the way of back-end changes, but what about bug fixes (if any appear) and UI updates? I have no desire to become the maintainer, but I'd consider doing a basic UI refresh and sending a pull request.

**Comment by nolanlawson** [comment]
@damccull No, I'm not maintaining it anymore. Pull requests are welcome, though &ndash; if you do a visual reboot, I'd be happy to publish it to the store. Feel free to increment the min SDK a well; I can just ship two APKs in the store.

**Comment by IriaSomobu** [comment]
Hi!

I've made some UI improvements (or editions? Changes?).
<details><summary>Click to see screenshots</summary>

![screenshot_20170224-225033](https://cloud.githubusercontent.com/assets/17745136/23318877/f020baf6-fadc-11e6-9a62-6b0b9559b9da.png)
![screenshot_20170224-225045](https://cloud.githubusercontent.com/assets/17745136/23318882/f4893aa0-fadc-11e6-8da8-6e47e74efab0.png)
![screenshot_20170224-225104](https://cloud.githubusercontent.com/assets/17745136/23318891/fa5fbb8e-fadc-11e6-813d-641c8bd6a60f.png)
![screenshot_20170224-225246](https://cloud.githubusercontent.com/assets/17745136/23318897/005a5184-fadd-11e6-8899-b57b2570bba4.png)

</details>

This is not final version. What do you think about it?
